starter-code/iteration3/iteration3/src/app/app.component.html

```diff
@@ -0,0 +1,13 @@
+<ul class="flex">
+  <li *ngFor="let oneChampionship of championship">
+    <h2>{{oneChampionship.year}}</h2>
+    <h3>{{oneChampionship.name}}</h3>
+    <img src={{oneChampionship.championUrl}}>
+    <h4>Best Players</h4>
+    <ul>
+      <li *ngFor="let oneList of list">
```

Comment:
Ok so you made a mistake here, actually you have to refer to the "list" key of the the oneChampionship object like this : 

```js

 <li *ngFor="let one of oneChampionship.list">

```